src/helper/selection-tools/reshape-tool.js

```diff
@@ -211,11 +254,22 @@ class ReshapeTool extends paper.Tool {
             this.mode = ReshapeModes.FILL;
             this._modeMap[this.mode].onMouseDown(hitProperties);
         }
-
-        // @todo Trigger selection changed. Update styles based on selection.
     }
     handleMouseMove (event) {
-        const hoveredItem = getHoveredItem(event, this.getHitOptions(), true /* subselect */);
+        const hitResult = this.getHitResult(event.point);
+        let hoveredItem;
+
+        if (hitResult) {
+            const item = hitResult.item;
+            if (item.selected) {
+                hoveredItem = null;
+            } else if (isBoundsItem(item)) {
```

Comment:
Looks like the reshape tool doesn't make use of grouping, so this if statement could be removed?

Reply:
I realized this is relevant for reshape after all, since rasters and text are bounds items (although you can't really reshape either of them)